apps/pxweb2/src/app/components/Content/Content.module.scss

```diff
@@ -20,4 +20,17 @@
   }
   background-color: var(--px-color-background-default);
   border-radius: var(--px-border-radius-xlarge);
+
```

Comment:
Should this, and a little further below, only be 1 blank line, for consistency with the rest of the file?